src/lib/services/client-metrics/last-seen/last-seen-mapper.ts

```diff
@@ -0,0 +1,40 @@
+import { Logger } from '../../../logger';
+import { IFeatureOverview } from '../../../types';
+import { IFeatureLastSeenResults } from './last-seen-read-model';
+
+export class LastSeenMapper {
+    mapToFeatures(
```

Comment:
Map results of reading into environments of incoming features to maintain todays data structure, might change in follow up iteration